core/tests/protractor_utils/LibraryPage.js

```diff
@@ -100,7 +100,8 @@ var LibraryPage = function() {
       '.protractor-test-exp-summary-tile-title'))).perform();
 
     await waitFor.elementToBeClickable(
-      addToPlayLaterListButton, 'Add to \'Play Later\' list Icon taking too long to load');
+      addToPlayLaterListButton,
+        'Add to \'Play Later\' list Icon taking too long to load');
```

Comment:
Why is the second line indented more?

```suggestion
      addToPlayLaterListButton,
      'Add to \'Play Later\' list Icon taking too long to load');
```

Reply:
I think it's correctly indented PTAL on the  file through 'Files changed' section. Thanks!

Reply:
I think you refer my previous commits to review the file. Thanks!

Reply:
Yep, you're right. Sorry about that!